<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Peer To Peer Demo</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="debugLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>280</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Console</string>
    </property>
   </widget>
   <widget class="QListWidget" name="debugListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>290</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>20</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QListWidget" name="blocksListWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>130</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="hostsListWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>130</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="blocksLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>130</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Blocks</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostsLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>130</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Logged Hosts</string>
    </property>
   </widget>
   <widget class="QListWidget" name="mainListWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>270</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>185</y>
      <width>270</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Main view</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
